test/test_blockchain.js

```diff
@@ -263,6 +275,16 @@ describe('testContractContent', function () {
     });
 });
 
+//test ContractData
+describe('testContractData', function () {
+    this.timeout(5000);
+    it('get the ContractData', async () =>{
+        let result = await testContractData(chain, test_payment_contract_id, state_index, data_key);
+        expect(result).to.not.be.empty;
```

Comment:
The non-empty result for getting the Contract Data is meaningless. Any Invalid key will get result for this API. We need parse JSON and check result.

Considering adding test case of check 'channelCreator' for payment channel contract:

```
Contract ID = "CEvtYbdBMjTbnkQpJN6sdBnsQwTmn6JeLci"
State index = 1
Channel ID = "GLNj5dzVE44nHVoc8TxTtmcMNxQ1W8UvJgtaLF5zXxpi"
```

The expected DB key will be

```
TeCHna5EaZZkLEgENHkKDgeQePTZ2h7Khvf13p8NW2WkCiAv
```

And expected address of channelCreator (`value` in response JSON) will be 

```
AUB7XrJ2zxqE7i93bMdXmKoCGkbJA7iUmBp
```